ecommers/lib/core/provider_models/home_provider_model.dart

```diff
@@ -33,24 +34,24 @@ class HomeProviderModel extends ProviderModelBase with ItemsLoadingNotifier {
   }
 
   Future _fetchCategories() async {
-    _categoryList = await services.categoryService.fetchCategoryList();
```

Comment:
the reason why I added services alias to the package is that paginator field from the dependency_services conflicts with the paginator from ItemsLoadingNotifier:)

Reply:
Seems now everything is ok, am I right?
Also maybe better to rename it accordingly?